Source/ee/Ee_SubSystem.h

```diff
@@ -52,8 +55,8 @@ namespace Ee
 		CDMAC						m_dmac;
 		CGIF						m_gif;
 		CSIF						m_sif;
-		CVpu						m_vpu0;
-		CVpu						m_vpu1;
+		CVpu*						m_vpu0;
```

Comment:
It would probably be better to use a shared_ptr or unique_ptr for m_vpu0 and m_vpu1. I'd rather avoid any explicit memory management because I think it's error prone. You can use a typedef to make the type name shorter, something like `typedef std::shared_ptr<CVpu*> VpuPtr`.